app/graph/types/like_type.rb

```diff
@@ -2,6 +2,8 @@
   name 'Like'
   description 'An association between a user and piece of content that represents the user liking the content.'
 
-  field :user, !UserType
+  field :user, UserType
```

Comment:
Should this be nullable?

Reply:
yup.

Reply:
Ops, I misread.
It should be non-nullable.